labcontrol/gui/templates/plate.html

```diff
@@ -159,31 +159,46 @@ <h4>Plate map</h4>
 
     <div id='plate-map-div' class='spreadsheet-container'></div>
 
-    <!-- Plate map legend div -->
+    <!-- Plate map legend div
+        Note that well-unknown and well-prev-plated are mutually exclusive
+        while well-duplicated is not. A duplicated value can appear in the
+        plate and be valid, unknown, or previously plated on another plate. -->
     <div id='plate-map-legend'><h5>Plate map legend</h5>
         <div id='legend-rows' class='legend-container'>
           <section id='plate-map-legend-commented' class='legend-entry'>
-            <div id='plate-map-legend-commented-box' class='well-commented legend-patch'></div>
+            <div id='plate-map-legend-commented-box' class='well-commented legend-patch'><center>Specimen ID</center></div>
```

Comment:
I know that this text being centered isn't critical, but the `<center>` tag is [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center)—we can't guarantee it'll be supported by modern browsers. A better solution would just be adding a CSS rule that says

```css
#plate-map-legend-commented-box {
    text-align: center;
}
```

...or something like that.

Reply:
Ha! I knew you would catch that! Thanks for the replacement!